pywbem/_subscription_manager.py

```diff
@@ -343,30 +343,38 @@ def add_server(self, server):
         self._owned_subscriptions[server_id] = []
         self._owned_filters[server_id] = []
         self._owned_destinations[server_id] = []
+        # self._systemnames[server_id] = None
+
+        # TODO expand this to account for WBEM Server profile.
```

Comment:
Based on the comments below, I think we no longer need to remember the SystemName properties, i.e. remove `self._systemnames`.

Reply:
I have no problem removing the TODO since that was a leftover from a much earlier pr.